doc/changes/latest.rst

```diff
@@ -13,126 +15,152 @@ HIGHLIGHTS
  | **Support for Python 3.8 has been dropped.**
  | **We recommend upgrading to Python 3.11 or above.**
  |
+ | **This version is the first one to support Python 3.13.**
+ |
  | **Minimum supported versions of the following packages have been bumped up:**
  | - numpy -- 1.22.4
  | - nibabel -- 5.2.0
  | - scikit-learn -- 1.4.0
  | - joblib -- 1.2.0
  | - pandas -- 2.2.0
 
-NEW
----
+-   This version includes a new "surface" API that allow to represent surface meshes and data.
+
+-   Single hemisphere meshes can be instantiated with
+    :class:`~nilearn.surface.SurfaceMesh`, :class:`~nilearn.surface.InMemoryMesh` or :class:`~nilearn.surface.FileMesh`.
```

Comment:
Last time I tried SurfaceMesh could not be used to instantiate a mesh. It does not take any arguments.

I think users should rely on `load_surf_mesh` to load single hemisphere meshes if they are loading it from a file.

InMemoryMesh is only useful if people are defining a mesh on their own (would be a pretty rare case IMO)

Reply:
OK so maybe best to remove this entire section to avoid confusion.